lib/interceptor/response-error.js

```diff
@@ -16,7 +16,7 @@ class ResponseErrorHandler extends DecoratorHandler {
   }
 
   #checkContentType (contentType) {
-    return this.#contentType.indexOf(contentType) === 0
+    return (this.#contentType ?? '').indexOf(contentType) === 0
```

Comment:
This looks fishy; more like a side-effect of not handling responses without `content-type` correctly.

Shall we allow them to be cached if we do not know the type?

cc: @ronag @flakey5 @mcollina

Reply:
I don't think there is any limitation of content types in caching.